benchmarks/cudf-merge.py

```diff
@@ -80,7 +92,12 @@ async def exchange_and_concat_bins(rank, eps, bins, timings=None):
     if timings is not None:
         t2 = clock()
         timings.append(
-            (t2 - t1, sum([sys.getsizeof(b) for i, b in enumerate(bins) if i != rank]))
+            (
+                t2 - t1,
+                sum(
+                    [sizeof_cudf_dataframe(b) for i, b in enumerate(bins) if i != rank]
```

Comment:
So we're measuring the amount of data we've received here?

Reply:
That is correct.